spec/requests/v0/backend_status_spec.rb

```diff
@@ -26,6 +28,8 @@
     end
 
     context 'for the gibs service' do
+      before { Flipper.disable(:sob_updated_design) }
```

Comment:
BackendServices::GI_BILL_STATUS was removed in the backend_statuses_controller. Can you please remove traces from this spec?